api/api.proto

```diff
@@ -281,6 +281,28 @@ message IoStrategy {
   bool early_ack = 2;
 }
 
+// ExportProtocol defines how the device is exported..
+enum ExportProtocol  {
+  // Invalid uninitialized value
+  INVALID = 0;
+  // PXD the volume is exported over Portworx block interace.
+  PXD = 1;
+  // ISCSI the volume is exported over ISCSI.
+  ISCSI = 2;
+  // NFS the volume is exported over NFS.
+  NFS = 3;
+  // Custom the volume is exported over custom interace.
+  CUSTOM = 4;
+}
+
+// ExportSpec defines how the volume is exported..
+message ExportSpec {
+   // ExportProtocol defines how the volume is exported.
+   ExportProtocol export_protocol = 1;
+   // ExportOptions options exporting the volume.
+   string export_options = 2;
```

Comment:
Are these comma separated options ? Why not use map?

Reply:
What are some examples of options?

Reply:
The export string will be appended as-is to the default export string e..g "security_label"